memote/support/basic.py

```diff
@@ -220,6 +220,16 @@ def find_pure_metabolic_reactions(model):
     return set(model.reactions) - (exchanges | transporters | biomass)
 
 
+def is_constrained_reaction(rxn):
+    """Return whether a reaction has fixed constraints."""
+    if rxn.reversibility:
+        return True if (
```

Comment:
You can simply return the evaluated expression ;)
```python
return (rxn.lower_bound > -1000 or rxn.upper_bound < 1000)
```

Reply:
Ah you're right! I'll have it fixed first thing tomorrow morning!